app/cdap/components/SnackbarComponent/Components/TransitionComponent/index.tsx

```diff
@@ -33,6 +33,7 @@ export default function TransitionComponent(props) {
           tabIndex={0}
           className={classes.dismissSpan}
           onClick={() => props.close()}
+          data-testid="snackbar-close-button"
         >
           Dismiss
```

Comment:
Replace hardcoded texts with translation
Dismiss should come from translation file

Reply:
Done